frr.conf

```diff
@@ -0,0 +1,53 @@
+./configure.py -r frr | tee /etc/frr/frr.conf
```

Comment:
I see this frr.conf file as very opinionated. It enables some debug messages, creates one BGP router and one BFD profile each with specific configuration values. For example:
- assumes the user wants to redistribute all connected IP prefixes
- random (?) BFD timeout values
- defines a BGP neighbor group with support only for a single peer
- BGP neighbor group enables BFD but the defined BFP profile only and specifically applies to one peer rather than the group.

FRR is a complex routing protocol suite. I'm thinking users would be better off bringing their own configuration (frr.conf and daemons files) via a volume.